src/cl/teku/teku_launcher.star

```diff
@@ -257,7 +257,7 @@ def get_config(
         "--p2p-subscribe-all-subnets-enabled=true",
         "--p2p-peer-lower-bound={0}".format(MIN_PEERS),
         "--p2p-advertised-ip=" + PRIVATE_IP_ADDRESS_PLACEHOLDER,
-        "--p2p-discovery-site-local-addresses-enabled",
+        "--p2p-discovery-site-local-addresses-enabled=true",
```

Comment:
this is just me explicitly labelling that flag as true

Reply:
Did this actually have any effect? If so, could be worth updating the documentation here - https://docs.teku.consensys.io/reference/cli#p2p-discovery-site-local-addresses-enabled

Reply:
this didn't, i was just confused whether its true or false so hardset it